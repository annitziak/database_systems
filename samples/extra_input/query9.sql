SELECT Student.A, SUM(Student.A*Student.B) FROM Student GROUP BY Student.A;
