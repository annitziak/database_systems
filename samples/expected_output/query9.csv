101, 5769
102, 9428
103, 8464
104, 729
